mesonbuild/modules/gnome.py

```diff
@@ -1104,11 +1104,11 @@ def _get_scanner_ldflags(ldflags: T.Iterable[str]) -> T.Iterable[str]:
         KwargInfo('includes', ContainerTypeInfo(list, (str, GirTarget)), default=[], listify=True),
         KwargInfo('install_gir', (bool, NoneType), since='0.61.0'),
         KwargInfo('install_dir_gir', (str, bool, NoneType),
-                  deprecated_values={False: ('0.61.0', 'Use install_gir to disable installation')},
+                  feature_validator=type_feature_deprecated_validator({False: ('0.61.0', 'Use install_gir to disable installation')}),
```

Comment:
Isn't false a value, not a type?

Reply:
Yeah, now that I check the CI logs this is causing failures across the board.

Reply:
I know I fixed this. Grrr